tools/report-converter/codechecker_report_converter/analyzer_result.py

```diff
@@ -72,6 +72,14 @@ def _add_report_hash(self, plist_objs):
 
     def _get_analyzer_result_file_content(self, result_file):
         """ Return the content of the given file. """
+        if not os.path.exists(result_file):
+            LOG.error("Result file does not exists: %s", result_file)
+            return
```

Comment:
This function may return `None` from now, but this is checked only for pyflakes report converter. Shouldn't this be checked at other analyzer tools' report converter at the caller side?

Reply:
Yes, good catch. I fixed it.